doc/source/cluster/kubernetes/examples/rayjob-kueue-priority-scheduling.md

```diff
@@ -16,68 +16,31 @@ Kueue has native support for some KubeRay APIs. Specifically, you can use Kueue
 to manage resources consumed by RayJob and RayCluster. See the
 [Kueue documentation](https://kueue.sigs.k8s.io/docs/overview/) to learn more.
 
-## Create a Kubernetes cluster on GKE
+## Step 0: Create a Kubernetes cluster on GKE (Optional)
 
 If you already have a Kubernetes cluster with GPUs, you can skip this step.
+Otherwise, follow [Start Google Cloud GKE Cluster with GPUs for KubeRay](kuberay-gke-gpu-cluster-setup) for more details on setting up a Kubernetes cluster on GKE.
 
-Create a GKE cluster with autoscaling enabled:
```

Comment:
Try to unify the instructions for setting up a K8s cluster. Having multiple K8s setup instructions will make it hard for us to maintain the documentation.